app-nia-catalog/build.gradle.kts

```diff
@@ -26,6 +28,22 @@ plugins {
 }
 
 kotlin {
+    @OptIn(ExperimentalWasmDsl::class)
+    wasmJs {
```

Comment:
I planned to add these pieces of code to the convention plugin, but I found that most of the libraries did not support wasm.